card/mod/pointer/set/abstract/02_pointer/html_views/radio_input.haml

```diff
@@ -11,4 +11,3 @@
         = radio_button_tag input_name, option_name, checked,
                            id: id, class: "pointer-radio-button #{'_submit-on-change' if submit_on_change}"
         = option_label option_label, id
-        = option_description "radio", option_name
```

Comment:
I don't like it that we make it impossible additional help. At least we should offer a view that gives access to additional help. For example with a question mark tooltip.

Reply:
I agree.  This gets quickly into sharking views...